src/atomate2_temp/aims/flows/phonons.py

```diff
@@ -0,0 +1,49 @@
+from dataclasses import dataclass, field
+from typing import TYPE_CHECKING
+
+from atomate2_temp.aims.jobs.base import BaseAimsMaker
+from atomate2_temp.aims.jobs.core import StaticMaker, RelaxMaker
+from atomate2_temp.aims.jobs.phonons import (
+    PhononDisplacementMaker,
+    PhononDisplacementMakerSocket,
+)
+from atomate2_temp.common.flows.phonons import BasePhononMaker
+
+
+@dataclass
+class PhononMaker(BasePhononMaker):
+    """
+    Maker to calculate harmonic phonons with VASP and Phonopy.
```

Comment:
That's the flow for aims, right? Then may be remove VASP reference from the docstring

Reply:
done